lms/djangoapps/discussion/static/discussion/js/spec/discussion_profile_page_factory_spec.js

```diff
@@ -15,10 +17,12 @@ define(
                     DiscussionProfilePageFactory(_.extend(
                         {
                             courseId: testCourseId,
-                            $el: $('.discussion-user-threads'),
-                            user_info: DiscussionSpecHelper.getTestUserInfo(),
                             roles: DiscussionSpecHelper.getTestRoleInfo(),
-                            sort_preference: null,
+                            courseSettings: DiscussionSpecHelper.createTestCourseSettings().attributes,
+                            $el: $('.discussion-user-threads'),
```

Comment:
It is strange that we are inconsistent in edx-platform between using ``el:`` and ``$el:``. The documentation recommends ``el:`` so I think we should change everything to use that:

http://backbonejs.org/#View-el